Setup assembly plugin

rexim opened the pull request:
<!-- NOTICE: this template serves as a recommendation, not as a requirement. -->
### Description

Extract fatjar config from #203 because of #208 

Close #45

rexim committed:
Setup assembly plugin (#45)

rexim committed:
Update README (#45)

ForNeVeR commented on README.md:
```diff
@@ -59,6 +59,10 @@ And after that you can take a look at the Unit Test coverage
 results. Just open `target/scala-2.11/scoverage-report/index.html`
 with your favorite browser.
 
+### Build an uber jar ###
```
I propose letter `ü` here because it's cool.

ForNeVeR commented on README.md:
```diff
@@ -59,6 +59,10 @@ And after that you can take a look at the Unit Test coverage
+### Build an uber jar ###
+
+    $ sbt assembly
+
```
Also please specify a location of the überjar produced.

rexim committed:
Improve README (#45)

ForNeVeR commented on README.md:
```diff
@@ -59,6 +59,14 @@ And after that you can take a look at the Unit Test coverage
 
+### Build an überjar ###
+
+To build an übérjàr run the following command
```
Ugh that's too hot!

rexim committed:
Use uberjar because it's search friendly (#45)

When it's called just uberjar it's easier to find it in some text
editors like Emacs. Unfortunately, they do not normolize diacritics as
the modern browser do.

rexim commented:
Thank you guys for the review!
